frontend/components/queries/ScheduledQueriesList/ScheduledQueriesListItem/ScheduledQueriesListItem.jsx

```diff
@@ -13,16 +13,20 @@ const baseClass = 'scheduled-query-list-item';
 const generatePlatformText = (platforms) => {
   const ALL_PLATFORMS = [
     { text: 'All', value: 'all' },
-    { text: 'macOS', value: 'darwin' },
     { text: 'Windows', value: 'windows' },
     { text: 'Linux', value: 'linux' },
+    { text: 'macOS', value: 'darwin' },
   ];
 
   if (platforms) {
-    const platformArray = platforms.split(',');
+    const platformsArray = platforms.split(',');
+
+    const textArray = platformsArray.map((platform) => {
+      // Trim spaces from the platform
+      const trimmedPlatform = platform.trim();
 
-    const textArray = platformArray.map((platform) => {
-      const text = find(ALL_PLATFORMS, { value: platform }).text;
+      // Convert value to text if value exists. Otherwise, just use the value.
+      const text = find(ALL_PLATFORMS, { value: trimmedPlatform }) ? find(ALL_PLATFORMS, { value: trimmedPlatform }).text : trimmedPlatform;
```

Comment:
Can you split this into two lines to only call `find` once? I think that will help both readability and performance (though `ALL_PLATFORMS` is a very small array so it won't really be significant for performance).